discord/message.py

```diff
@@ -1114,3 +1148,74 @@ async def ack(self):
         if state.is_bot:
             raise ClientException('Must not be a bot account to ack messages.')
         return await state.http.ack_message(self.channel.id, self.id)
+
+    async def reply(self, content=None, *, tts=False, embed=None, file=None,
+                                          files=None, delete_after=None, nonce=None,
+                                          allowed_mentions=None):
+        """|coro|
```

Comment:
this whole docstring can be simplified to refer to `Messageable.send`. E.G.: https://github.com/Rapptz/discord.py/blob/93fa46713a198baf08ab1e760be1adfdcb852c97/discord/channel.py#L838-L848

Reply:
Simplified call signature and inserted docs reference to `Messageable.send`.